bones/bone.py

```diff
@@ -119,6 +119,27 @@ def setSystemInitialized(self):
 		"""
 		pass
 
+	def isInvalid(self, value):
+		"""
+			Returns None if the value would be valid for
+			this bone, an error-message otherwise.
+		"""
+		return False
+
+	def isEmpty(self, rawVale: Any) -> bool:
+		"""
+			Check if the given single value represents the "empty" value.
+			This usually is the empty string, 0 or False.
+
+			Warning: isEmpty takes precedence over isInvalid! The empty value is always valid - unless the bone
+				is required. But even then the empty value will be reflected back to the client.
+
+			Warning: rawValue might be the string/object received from the user (untrusted input!) or the value
+				returned by get
+
+		"""
+		return not bool(rawVale)
```

Comment:
could you please fix the typo?
```suggestion
	def isEmpty(self, rawVaule: Any) -> bool:
		"""
			Check if the given single value represents the "empty" value.
			This usually is the empty string, 0 or False.

			Warning: isEmpty takes precedence over isInvalid! The empty value is always valid - unless the bone
				is required. But even then the empty value will be reflected back to the client.

			Warning: rawValue might be the string/object received from the user (untrusted input!) or the value
				returned by get

		"""
		return not bool(rawValue)
```